@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/electrical/fn_electricalController.sqf

```diff
@@ -18,8 +18,18 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli", "_deltaTime"];
 
+private _configVehicles = configFile >> "CfgVehicles" >> typeof _heli;
+private _flightModel    = getText (_configVehicles >> "flightModel");
+
 private _apuOn  = _heli getVariable "fza_systems_apuOn";
-private _rtrRPM = (_heli animationPhase "mainRotorRPM") / 10;
+private _rtrRPM = 0.0; 
```

Comment:
Please can the rotor rpm here be abstracted into its own function?

Reply:
Where would it reside?

Reply:
Completed, resolving